app/src/main/res/values-fr/strings.xml

```diff
@@ -2,39 +2,45 @@
 
     <!--Sections-->
     <string name="modules">Modules</string>
-    <string name="superuser">Super‑utilisateur</string>
+    <string name="superuser">Super-utilisateur</string>
     <string name="logs">Journaux</string>
     <string name="settings">Paramètres</string>
+    <string name="refresh">Rafrachir Données locales</string>
     <string name="install">Installer</string>
     <string name="section_home">Accueil</string>
     <string name="section_theme">Thèmes</string>
     <string name="safetynet">SafetyNet</string>
 
     <!--Home-->
     <string name="no_connection">Aucune connexion disponible</string>
-    <string name="home_notice_content">Assurez‑vous de toujours utiliser la version libre de Magisk Manager. Une version provenant d’une source inconnue peut effectuer des actions malveillantes.</string>
-    <string name="hide">Masquage</string>
-    <string name="app_changelog">Journal des modifications</string>
-    <string name="manager">Le gestionnaire</string>
+    <string name="app_changelog">Journal des changements</string>
+    <string name="manager">Gestionnaire</string>
+    <string name="loading">Chargement…</string>
+    <string name="info">Info</string>
+    <string name="uninstall">Désinstallation</string>
+    <string name="update">Mise à jour</string>
+    <string name="not_available">N/A</string>
+    <string name="hide">Masquer</string>
+    <string name="status">Statut</string>
+    <string name="home_package">Paquet</string>
+    <string name="details">Détails</string>
 
-    <string name="home_support_title">Soutenez‑nous</string>
+    <string name="home_notice_content">Soyez sûr d\’utiliser le Gestionnaire Magisk open-source. Les Gestionnaires d\’origine inconnue peuvent éffectuer des actions malintentionnées.</string>
+    <string name="home_support_title">Soutenez-nous</string>
     <string name="home_links_project">Liens du projet</string>
     <string name="home_item_source">Sources</string>
-    <string name="home_support_content">Magisk est, et sera toujours, libre et open‑source. Vous pouvez cependant nous témoigner de votre soutien en envoyant un petit don.</string>
-
-    <string name="home_device_system">Système</string>
-
-    <string name="home_package">Paquet</string>
-
+    <string name="home_support_content">Magisk est, et sera toujours, libre et open-source. Vous pouvez cependant nous témoigner de votre soutien en envoyant un petit don.</string>
+    <string name="home_device_title">Périphérique</string>
+    <string name="home_device_extra_manufacturer">Fabriquant</string>
+    <string name="home_device_extra_model">Modèle</string>
+    <string name="home_device_extra_board">Adminitstration</string>
```

Comment:
I'm pretty sure "Adminitstration" is wrong in this context.
It's more likely "carte système"/"carte d'appareil" or just "plateforme".
Polish use "platforma" and Spanish "placa".

P.S.: Romanian maintainer here, not sure what translation to add, but I'm inclined to use "platform" for my language.

Reply:
As i don't know exactly where is this word, i have set 'Plateforme' instead of 'Adminitstration', as i can't be sure, for now i left it like that.
More fix(s) will come if needed 👍

Reply:
![Screenshot_20200211-214545_Magisk_Manager](https://user-images.githubusercontent.com/2406819/74272882-12e87700-4d18-11ea-9ca3-f02e07bd7026.png)

Have a look at this image, in my case "msm8996" is the board. Which is the Snapdragon 820/821 platform.

My French skills are pretty limited, but wanted to see how other languages (will) translate that "Board" string and noticed "Adminitstration" in your pull request and wrote here the above :)

Reply:
Aaahhh okkkk sorry my bad didn't see that before, so in French it's 'Socle', i don't know another more appropriated translation.